prototype2/multiblade/generators/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#=============================================================================
+# Build MB text 2 UDP
+#=============================================================================
+set(mbtext2udp_INC Args.h)
+set(mbtext2udp_SRC text2udp.cpp Args.cpp TextFile.cpp)
+create_executable(mbtext2udp)
```

Comment:
mbtext2udp can be deleted. It no longer produces the correct readout format and
I prefer to use pcap files directly.